javascripts/tagmanagerHelper.js

```diff
@@ -173,9 +174,36 @@
         var params = {method: 'TagManager.disablePreviewMode', idContainer: idContainer};
         piwikHelper.modalConfirm('<h2>' + _pk_translate('TagManager_DisablingPreviewPleaseWait') + '</h2>', {});
         piwikApi.fetch(params).then(function () {
+            tagManagerHelper.updateDebugSiteFlag(piwik.siteMainUrl, idContainer, -1);
             window.location.reload();
         });
     };
+    tagManagerHelper.changeDebugUrl = function (idContainer, oldUrl) {
+        var newUrl = document.getElementById('previewDebugUrl').value;
+        var id = 'TagManager_changeDebugSiteUrl';
+        var context = 'warning'; // or 'warning' or 'error' or 'success'
+        var UI = require('piwik/UI');
+        var notification = new UI.Notification();
+        if (!newUrl) {
+            return notification.show(_pk_translate('TagManager_DebugUrlNoUrlErrorMessage'), {context: context, id: id, title: ''});
+        } else if (newUrl === oldUrl) {
+            return notification.show(_pk_translate('TagManager_DebugUrlSameUrlErrorMessage', ['<a href="'+newUrl+'" target="_blank">','</a>']), {context: context, id: id, title: ''});
+        }
+        var piwikApi = piwikHelper.getAngularDependency('piwikApi');
```

Comment:
fyi usually when we're outside angular then we use the `ajaxHelper` class. It be usually fine to use `piwikApi` but as we're also in the process of removing angular it may be better to use `ajaxhelper` here so we won't need to migrate it later.

Like 

```

            var ajaxRequest = new ajaxHelper();
            ajaxRequest.addParams(ajaxParams, 'get');
            ajaxRequest.withTokenInUrl();
            ajaxRequest.setCallback(callback);
            ajaxRequest.setFormat('html');
            ajaxRequest.send();
```

Reply:
started using ajaxHelper